regexes/client/browsers.yml

```diff
@@ -5,6 +5,11 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# SFive (https://sfive.vn/)
+- regex: 'SFive/(?:(\d+[\.\d]+))?'
```

Comment:
default engine: Blink for Linux/Win/Android

and detect for IOS, engine webkit

* [ ] Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_3 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) SFive_iOS/1.0.0.16 CriOS/57.0.2979.0 Mobile/14G60 Safari/602.1

and detect for Android, engine Blink
* [ ] Mozilla/5.0 (Linux; Android 8.0.0; SM-G935F Build/R16NW) AppleWebKit/537.36 (KHTML, like Gecko) SFive_Android/65.0.3325.109 Chrome/65.0.3325.109 Mobile Safari/537.36


```yml
- regex: 'SFive(?:_Android)?/.+ Chrome/(\d+[\.\d]+)'
  name: 'SFive'
  version: '$1'
  engine:
    default: 'Blink'
- regex: 'SFive_IOS/(\d+[\.\d]+)'
  name: 'SFive'
  version: '$1'
```